open-metadata-implementation/user-interfaces/presentation-server/cra-server/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cra-server",
+  "version": "1.0.0",
+  "description": "Egeria Presentation Server",
+  "main": "index.js",
+  "scripts": {
+    "start": "run-p start:server start:client",
+    "start:server": "nodemon index.js",
+    "start:client": "cd ../cra-client && npm start",
+    "build": "cd ../cra-client && npm run build",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Jacob Madsen",
```

Comment:
We should probably list it as 'Egeria contributors' or similar, we we do with the copyright

Reply:
Agreed, updated both package.json files

Reply:
thanks